### DISCUSSION TITLE
zh/nix-store/add-binary-cache-servers

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#166] [Announcements]
# zh/nix-store/add-binary-cache-servers

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/nix-store/add-binary-cache-servers

**Comment by JianleiZhang** [comment]
使用普通用户写入override.conf文件时，可能会提示权限错误，需要改成以下方式写入。
```
sudo bash -c 'cat << EOF >/run/systemd/system/nix-daemon.service.d/override.conf
[Service]
Environment="https_proxy=socks5h://localhost:7891"
EOF'
```

**Comment by DualHappiness** [comment]
为何我使用 nixos-wsl 可以实现shell 改proxy 同时生效？24.05版本的
